### PR TITLE
feat: 'add to team' => 'install bot' on bot profiles

### PR DESCRIPTION
**Pull request by hyperobject**
This PR changes the Actions component on user profiles to replace the "Add to team..." button with an "Install bot in team or chat" button in the ... menu for featured bot profiles. I don't feel great about having to load the featured bots on first render, but there's not currently a better way to check if a bot is in our featured bots list.

### CODE REVIEW

File: shared/profile/user/actions/index.tsx [hunk 1]
```diff
@@ -151,7 +163,9 @@ const Actions = (p: Props) => {
 
 const DropdownButton = Kb.OverlayParentHOC((p: Kb.PropsWithOverlay<DropdownProps>) => {
   const items: Kb.MenuItems = [
-    {icon: 'iconfont-people', onClick: p.onAddToTeam, title: 'Add to team...'},
+    p.isBot
+      ? {icon: 'iconfont-nav-2-robot', onClick: p.onInstallBot, title: 'Install bot in team or chat'}
```
Comment (jakob223) [inline shared/profile/user/actions/index.tsx]: 🤖

**Comment by hyperobject** [comment]
cc @mmaxim thoughts about where we should load featured bots for non-chat stuff? might be good to do the same thing for the team picker since i'm using it here as well

**Comment by mmaxim** [comment]
Should be fine, they are cached in the service, so this call should always be fast (at least as fast as all the other stuff that gets loaded on profiles).

**Comment by jakob223** [comment]
Then LGTM

**Pull request merged**